src/extension/background-script/nostr/index.ts

```diff
@@ -59,6 +60,31 @@ class Nostr {
     return Utf8.stringify(decrypted);
   }
 
+  async delegate(delegateePubkey: string, conditions: Nip26DelegateConditions) {
+    const cond = [];
+
+    if (conditions.kind) cond.push(`kind=${conditions.kind}`);
+    if (conditions.until) cond.push(`created_at>${conditions.until}`);
+    if (conditions.since) cond.push(`created_at<${conditions.since}`);
+    if (conditions.content) cond.push(`content=${sha256(conditions.content)}`);
```

Comment:
Is this a standard nip26 condition? Does not seems to be defined as option [among NIP-26 conditions](https://github.com/nostr-protocol/nips/blob/master/26.md#conditions-query-string)

I do think if we want to restrict the content the delegatee can sign, we should also adding to the NIP?

Reply:
under review, but wanted to get ahead of it to show how it works: https://github.com/nostr-protocol/nips/pull/222